src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -14,6 +19,10 @@ public partial class DocumentationBrowserView : UserControl, IDisposable
     {
         private const string ABOUT_BLANK_URI = "about:blank";
         private readonly DocumentationBrowserViewModel viewModel;
+        private const string FALLBACK_DOC_DIRECTORY_NAME = "fallback_docs";
```

Comment:
isn't this string already defined somewhere in this project?

Reply:
@mjkkirschner yes, is defined in PackageDocumentationManager.cs and DocumentationBrowserViewExtension.cs (I just move it here).
Do you have any idea in which place I can place it so we can use it in both places?